conn.go

```diff
@@ -36,12 +37,31 @@ func (d *Driver) Open(dsn string) (driver.Conn, error) {
 	return &Conn{h: h}, nil
 }
 
-func (c *Conn) Close() error {
+func (c *Conn) Close() (err error) {
+	if c.tx != nil {
```

Comment:
Why are you changing (*Conn).Close? Wouldn't ODBC rollback transaction if you release handle without committing it?